config-schema.json

```diff
@@ -727,16 +760,16 @@
             "queueSize": {
               "$id": "#/properties/settings/properties/logging/properties/queueSize",
               "type": "integer",
-              "minimum": 0,
               "title": "Log message queue size for websocket API",
-              "default": 100
+              "default": 0,
+              "minimum": 0
             },
             "purgeHours": {
               "$id": "#/properties/settings/properties/logging/properties/purgeHours",
               "type": "integer",
-              "minimum": 0,
               "title": "Purge log files in hours",
-              "default": 12
+              "default": 0,
```

Comment:
Attention: does this disable the log file purging or does it immediately delete the log files whenever it checks if files need to be deleted?
Behaviour should be documented :-)